cgi/members/memberentry.pl

```diff
@@ -605,7 +605,8 @@ BEGIN
 #            -default => $default,
 #        );
 my $CGIbranch = [];
-foreach(keys %$branches) {
+# Sort branches by branch name before generating CGI branch array
+foreach (sort {lc(%$branches{$a}{branchname}) cmp lc(%$branches{$b}{branchname})} keys %$branches) {
```

Comment:
I'm a little new to Perl, so thanks for the correction! I'll try again using what you wrote.